partcad/src/partcad/project.py

```
@@ -123,8 +128,20 @@
         def __exit__(self, *_args):
             self.lock.release()
 
-    def __init__(self, ctx, name, path, include_paths=[]):
-        super().__init__(name, path, include_paths)
+    def __init__(
+        self,
+        ctx,
+        name: str,
+        path: str,
+        include_paths: list[str] = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)